contracts/wallet/WalletFactory.sol

```diff
@@ -196,7 +136,10 @@ contract WalletFactory is Owned, Managed {
         returns (address _wallet)
     {
         require(_guardian != (address(0)), "WF: guardian cannot be null");
```

Comment:
To be consistent we should add a check that `_owner != address(0)` and that `_modules.length > 0` as we know we cannot create wallets without these 2 conditions.